src/ok.rs

```diff
@@ -51,13 +55,220 @@ impl fmt::Display for TestState {
 }
 
 fn collect_tests() -> Vec<Test> {
-    vec![]
+    let tests = vec![Test {
+        name: "AMD CPU",
+        gen_mask: SEV_MASK,
+        run: Box::new(|| {
+            let res = unsafe { x86_64::__cpuid(0x00000000) };
+            let name: [u8; 12] = unsafe { transmute([res.ebx, res.edx, res.ecx]) };
+            let name = from_utf8(&name[..]).unwrap();
```

Comment:
No panicking. Just unwrap_or_else(|| "???".to_string()) or something to that effect. As long as you don't give it the default value of `AuthenticAMD`, the test will fail as we want it to.